extensions/replication/queueProcessor/task.js

```diff
@@ -44,9 +45,13 @@ metricsProducer.setupProducer(err => {
                 destConfig.bootstrapList = config.getBootstrapList();
             });
 
-            const queueProcessor = new QueueProcessor(zkConfig, kafkaConfig,
-                sourceConfig, destConfig, repConfig, metricsProducer);
-            queueProcessor.start();
+            // Start QueueProcessor for each site
+            const siteNames = bootstrapList.map(i => i.site);
+            siteNames.forEach(site => {
+                const queueProcessor = new QueueProcessor(zkConfig, kafkaConfig,
```

Comment:
You cannot define a constant in a loop.

Reply:
shouldn't `const` be used here? We are only using within the scope of the loop and using `let` will throw a linter error since the variable is never redefined